app/controllers/application_controller.rb

```diff
@@ -10,6 +10,10 @@ class ApplicationController < ActionController::Base
   around_action :set_time_zone, if: :current_user
   layout 'main'
 
+  rescue_from ActionController::InvalidAuthenticityToken do
+    redirect_to root_path
+  end
```

Comment:
Should we somehow notify user?

Reply:
I don't think we can display some nice message here and not confuse users